pkg/server/helpers.go

```diff
@@ -51,46 +49,24 @@ const (
 	errorStartReason = "StartError"
 	// errorStartExitCode is the exit code when fails to start container.
 	// 128 is the same with Docker's behavior.
+	// TODO(windows): Figure out what should be used for windows.
 	errorStartExitCode = 128
```

Comment:
Is this for "command not exist"?

Reply:
Usually this is caused by command not found.

See this https://github.com/moby/moby/blob/master/daemon/start.go#L127